Gems/EditorPythonBindings/Code/Source/PythonLogSymbolsComponent.cpp

```diff
@@ -206,12 +212,12 @@ namespace EditorPythonBindings
         AZ::IO::FileIOBase::GetInstance()->Write(handle, buffer.c_str(), buffer.size());
     }
 
-    void PythonLogSymbolsComponent::LogClass(AZStd::string_view moduleName, AZ::BehaviorClass* behaviorClass)
+    void PythonLogSymbolsComponent::LogClass(AZStd::string moduleName, AZ::BehaviorClass* behaviorClass)
```

Comment:
this should probably be a const reference, same with the other calls below that just pass through to another function

Reply:
maybe, as long as the QueueBroadcast() can capture the arguments since the system now needs to hold onto the strings so that they can "execute queue" latter on. 

I will try this out.

Reply:
Yeah I think it makes sense for the deeper call to not be a reference, but most of those were just passing through to other functions it looked like.

Reply:
The calls not as deep in the call stack shouldn't use `const AZStd::string`, that prevents moving of those parameters when making deeper calls